11/jdk/alpine/Dockerfile.hotspot.releases.full.maven

```diff
@@ -0,0 +1,21 @@
+
+FROM docker.tivo.com/openjdk11:alpine
+
+ARG MAVEN_VERSION=3.6.1
+ARG USER_HOME_DIR="/root"
+ARG SHA=b4880fb7a3d81edd190a029440cdf17f308621af68475a4fe976296e71ff4a4b546dd6d8a58aaafba334d309cc11e638c52808a4b0e818fc0fd544226d952544
+ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+
+RUN apk add --update --no-cache wget curl bash && \
+    mkdir -p /usr/share/maven && \
+    curl -fsSL -o /tmp/apache-maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz && \
+    tar -xzf /tmp/apache-maven.tar.gz -C /usr/share/maven --strip-components=1 && \
+    rm -f /tmp/apache-maven.tar.gz && \
+    ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+
+ENV JAVA_HOME=/opt/java/openjdk \
+    MAVEN_HOME=/usr/share/maven \
+    MAVEN_CONFIG="$USER_HOME_DIR/.m2" \
```

Comment:
Don't hardcode/re-define/dupe home, re-use the already-declared var `$HOME`:

```
/ # export
export HOME='/root'
export HOSTNAME='626c441363b3'
export JAVA_HOME='/opt/java/openjdk'
export JAVA_VERSION='jdk-11.0.4+11'
export LANG='en_US.UTF-8'
export LANGUAGE='en_US:en'
export LC_ALL='en_US.UTF-8'
export PATH='/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
export PWD='/'
export SHLVL='1'
export TERM='xterm'

```

Reply:
Fixed now